region/crateria/central.json

```diff
@@ -200,92 +238,139 @@
+                  "notable": false,
+                  "requires": [
                     {"canShineCharge": {
                       "usedTiles": 19,
                       "steepUpTiles": 2,
                       "steepDownTiles": 1,
                       "shinesparkFrames": 45,
                       "openEnd": 2
-                    }},
-                    "canFly"
+                    }}
                   ]
                 }
               ]
             },
             {
               "id": 5,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             }
           ]
         },
         {
           "from": 5,
           "to": [
-            {
-              "id": 1,
-              "requires": [
-                {"and": [
-                  "canDestroyBombWalls",
-                  {"or": [
-                    "canFly",
-                    "canTrickyWalljump",
-                    "SpeedBooster",
-                    {"and": [
-                      "HiJump",
-                      "canSpringBallJumpMidAir"
-                    ]}
-                  ]}
-                ]}
-              ]
-            },
             {
               "id": 2,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 3,
-              "requires": [
-                {"canShineCharge": {
-                  "usedTiles": 31,
-                  "shinesparkFrames": 75,
-                  "openEnd": 2
-                }}
+              "strats": [
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    {"canShineCharge": {
+                      "usedTiles": 31,
+                      "shinesparkFrames": 75,
+                      "openEnd": 2
+                    }}
+                  ]
+                }
               ],
               "note": "This link is only for sparking. All other strats should go 5 -> 4 -> 3."
             },
             {
               "id": 4,
-              "requires": [
+              "strats": [
                 {
-                  "or": [
-                    "canWalljump",
-                    "canFly",
-                    "SpeedBooster",
-                    "canDestroyBombWalls",
-                    "HiJump"
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canWalljump",
+                      "canFly",
+                      "SpeedBooster",
+                      "canDestroyBombWalls",
+                      "HiJump"
```

Comment:
springball jump

Reply:
This is why trying to have walljump as a tech is so much work.